src/mkdocs_autorefs/plugin.py

```diff
@@ -83,26 +103,9 @@ def get_item_url(  # noqa: WPS234
 
         Returns:
             A site-relative URL.
-
-        Raises:
-            KeyError: If there isn't an item by this identifier anywhere on the site.
         """
-        try:
-            url = self._url_map[identifier]
-        except KeyError:
-            if identifier in self._abs_url_map:
-                return self._abs_url_map[identifier]
-
-            if fallback:
-                new_identifiers = fallback(identifier)
-                for new_identifier in new_identifiers:
-                    with contextlib.suppress(KeyError):
-                        url = self.get_item_url(new_identifier, from_url)
-                        self._url_map[identifier] = url  # update the map to avoid doing all this again
-                        return url
-            raise
-
-        if from_url is not None:
+        url = self._get_item_url(identifier, fallback)
+        if from_url is not None and not urlparse(url).scheme:
```

Comment:
```suggestion
        if from_url is not None:
            parsed = urlsplit(link)
            if not parsed.scheme and not parsed.netloc:
```
references:
* https://github.com/mkdocs/mkdocs/blob/dc35569ade5e101c8c48b78bbed8708dd0b1b49a/mkdocs/structure/pages.py#L205
* https://github.com/mkdocs/mkdocs/blob/1da44cea96b850c527d7bdceac920498e4a488ae/mkdocs/structure/nav.py#L128
* https://github.com/oprypin/mkdocs-literate-nav/blob/7544a156d8e9aad350128d13e3ee8ac1574b6c5f/mkdocs_literate_nav/parser.py#L109

Reply:
Thanks!